operators/gpu-operator-certified/v23.9.2/manifests/gpu-operator-certified.clusterserviceversion.yaml

```diff
@@ -0,0 +1,966 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  labels:
+    operatorframework.io/arch.arm64: supported
+    operatorframework.io/arch.amd64: supported
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/warn: privileged
+  annotations:
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/proxy-aware: "true"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "false"
+    features.operators.openshift.io/cni: "false"
```

Comment:
This is a duplicate...should be `csi`

Reply:
I think fixing this resolves both failures, since the digest pinning is also complaining about a duplicate key.